### DISCUSSION TITLE
How to trigger a Custom page?

### DISCUSSION THREAD

**Discussion by som45oul** [gickowtf/pixoo-homeassistant#7] [Q&A] [answered]
Hi there. I'm new to both the Pixoo and HA.

I can see you have provided a way to display a custom message via a trigger ... but I am wondering how to change the display to a Custom one using a trigger?  Is this possible?  Or have I misunderstood something?

Many thanks for your kind work on publishing this integration. Much appreciated!

**Comment by gickowtf** [comment, marked as answer]
I think there is a misunderstanding. You can only trigger a specially created page as a push.

Thank you very much for the kind words, I hope that I will find the time very soon to bring out a small update.

**Reply by som45oul** [reply]
Thanks for the quick response. That is a shame. Is this something you could potentially look at in future, perhaps?  I want to automate the change of display (Cloud, Custom 1, Custom 2, Customer 3, etc) via a Stream Deck via Home Assistant.

**Reply by gickowtf** [reply]
Yes, i put your idea on my todo 😉
